x-pack/extras/plugins/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+configurations {
+  allPlugins
```

Comment:
We need to reference this configuration in our cloud-ess docker image build. Next to https://github.com/elastic/elasticsearch/blob/38c90ca8d47001002a4c6c9bf34858088068d8c6/distribution/docker/build.gradle#L119 you need to add `allPlugins project(path: ':x-pack:extras:plugins', configuration: 'allPlugins')` so we take those plugins into account when packaging the cloud images